horovod/runner/launch.py

```diff
@@ -426,10 +453,23 @@ def parse_args():
                                choices=config_parser.LOG_LEVELS,
                                help='Minimum level to log to stderr from the Horovod backend. (default: WARNING).')
     group_logging_timestamp = group_logging.add_mutually_exclusive_group()
-    group_logging_timestamp.add_argument('--log-hide-timestamp', action=make_override_true_action(override_args),
-                                         help='Hide the timestamp from Horovod log messages.')
-    group_logging_timestamp.add_argument('--no-log-hide-timestamp', dest='log_hide_timestamp',
-                                         action=make_override_false_action(override_args), help=argparse.SUPPRESS)
+    group_logging_timestamp.add_argument('--log-with-timestamp', 
+                                         action=make_override_true_action(override_args),
+                                         help=argparse.SUPPRESS)
+    group_logging_timestamp.add_argument('--log-without-timestamp', dest='log_with_timestamp',
+                                         action=make_override_false_action(override_args), 
+                                         help='Hide the timestamp from Horovod internal log messages.')
+    group_logging_timestamp.add_argument('-prefix-timestamp', '--prefix-output-with-timestamp', action='store_true',
+                                         dest='prefix_output_with_timestamp',
+                                         help='Timestamp each line of output to stdout, stderr, and stddiag.')
+    group_logging_timestamp.add_argument('--log-hide-timestamp', 
+                                         dest='log_with_timestamp',
```

Comment:
I think this should read `log_without_timestamp`

Reply:
Good catch!

Reply:
Ah, actually this one is correct.  This is for the deprecated arg, so we avoid breaking existing users.  When they use this flag, they will get a deprecation warning.  We will remove this in the next major release.

Reply:
it is confusing though

Reply:
The args are suppressed from `horovodrun --help`, so only users who are already using them will be aware of them.